### PR TITLE
Add tamzin story

### PR DESCRIPTION
**Pull request by SarahAlidoost**
Slides are available in EUCP sharepoint `Tamzin_EUCP_stpryboard.pptx`

### CODE REVIEW

File: static/stories/sub_select.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+In many applications of ensemble climate projections such as CMIP6 it is
+impractical to apply realizations from all 30-50 members, and a smaller subset is
+required. Such applications might include climate change impacts studies (for
+example hydrological models to explore flood impacts) or dynamical downscaling
+with regional climate models.
+:::
+
+:::Chapter{headline="Processed based assessment criteria" image="example_assessment.png"}
+## Processed based assessment criteria
```
Comment (Peter9192) [inline static/stories/sub_select.md]: Should this be "process-based"?

File: static/stories/sub_select.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+Method builds on the approach developed in McSweeney et al.,(2015, 2018)
```
Comment (Peter9192) [inline static/stories/sub_select.md]: Can we add urls for these references?

File: static/stories/sub_select.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+- Seasonal  Temperature bias ​
+:::
+
+:::Chapter{headline="Performance scores" image="example_table.png"}
+
+**Performance Scores** of CMIP6 models for large scale and regional criteria ​
+
+The models were scored against 3 classifications to indicate their performance
+for each of the criteria. The classifications were defined for this assessment
+as follows: ​
```
Comment (Peter9192) [inline static/stories/sub_select.md:61]: I'm missing an explanation of the grey cells in the table.

File: static/stories/sub_select.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+:::Chapter{headline="Model independence​" image="clusters.png"}
+## Model Independence​
+
+18 models remain after filtering for performance. We can further sample these to
+offer smaller subsets (~5-7) which are still representative.​
+
+- Some models are closely related to each other due to shared components, the
+  degree of interdependence is measured using the ClimWIP method for the
+  comparison of model independence (Brunner et al., 2020). ​
```
Comment (Peter9192) [inline static/stories/sub_select.md]: Can we add a url for Brunner et al?

File: static/stories/sub_select.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+  - MRI-ESM2-0​
+  - MPI-ESM2-HR​
+:::
+
+:::Chapter{headline="Example model sub-selection" image="example_model_subselection.png"}
+## Example model sub-selection based on performance assessment and model independence.
+
+Example of model sub-selection for an ensemble of 5 -7 CMIP6 models (shown in
+blue) , as may be required for downscaling or hydrological assessments. ​
+
```
Comment (Peter9192) [inline static/stories/sub_select.md]:
```suggestion
Example of model sub-selection for an ensemble of 5 -7 CMIP6 models (shown in
blue), as may be required for downscaling or hydrological assessments. ​

```

File: static/stories/sub_select.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+Figure (left), shows where the selected models lie within the range of future
+(2081-2100) temperature and precipitation projections (ssp585). ​
+
+Models found be inadequate for large-scale assessment criteria are shown in red.
+​
+Models are removed (due to being classed inadequate) throughout the projection
```
Comment (Peter9192) [inline static/stories/sub_select.md]: Somehow the whitespace is not rendered in my local build. It reads like: "...shown in red.Models are..."

File: static/stories/sub_select.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+The information in the assessment table can be applied by users to create their
+own bespoke model sub-selection according their requirements. ​
+
+This could be used in combination with other tools such as
+[https://gcmeval.met.no](https://gcmeval.met.no) (see illustration on left), to
+pick models with acceptable performance in the required criteria that capture
+the ensemble range. ​
+
+Alternatively, we also present a list of models that is a sub-selection of the
+CMIP6 ensemble that was found to be acceptable for the main European regions. ​
```
Comment (Peter9192) [inline static/stories/sub_select.md]:
```suggestion
Alternatively, we also present a list of models that is a sub-selection of the
CMIP6 ensemble that was found to be acceptable for the main European regions: ​
```

**Review by Peter9192 — APPROVED** [review]
Thanks @SarahAlidoost for implementing! I'll ask Tamzin to provide answers to some of the comments as well.

File: static/stories/sub_select.md (hunk 1 above)
Comment (tepmo42) [reply static/stories/sub_select.md]: Process based assessment criteria .

File: static/stories/sub_select.md (hunk 2 above)
Comment (tepmo42) [reply static/stories/sub_select.md]: Yes, do you want me to send them?

File: static/stories/sub_select.md (hunk 7 above)
Comment (tepmo42) [reply static/stories/sub_select.md]: Yes.

File: static/stories/sub_select.md (hunk 2 above)
Comment (Peter9192) [reply static/stories/sub_select.md]: Please :-)

File: static/stories/sub_select.md (hunk 6 above)
Comment (SarahAlidoost) [reply static/stories/sub_select.md]: fixed.

File: static/stories/sub_select.md (hunk 1 above)
Comment (SarahAlidoost) [reply static/stories/sub_select.md]: fixed.

File: static/stories/sub_select.md (hunk 2 above)
Comment (tepmo42) [reply static/stories/sub_select.md]: https://doi.org/10.1007/s00382-014-2418-8 McSweeney et al 2015 - I can't find a url for 2018 as its a technical report.

File: static/stories/sub_select.md (hunk 4 above)
Comment (tepmo42) [reply static/stories/sub_select.md]: https://doi.org/10.5194/esd-11-995-2020

File: static/stories/sub_select.md (hunk 2 above)
Comment (SarahAlidoost) [reply static/stories/sub_select.md]: @tepmo42 thanks, could you please provide us the title of the technical report?

Comment (tepmo42) [reply static/stories/sub_select.md]: here is a citation:  Mcsweeney, C. F., Murphy, J., Sexton, D. M. H., Rostron, J. W., Yamazaki, K., & Harris, G. R. (2018). Selection of CMIP5 members to augment a perturbed-parameter ensemble of global realisations of future climate for the UKCP18 scenarios. Hadley Centre Technical Note. 102

Comment (SarahAlidoost) [reply static/stories/sub_select.md]: @tepmo42 great, thanks. I found this [link](https://digital.nmla.metoffice.gov.uk/IO_28f93601-3178-4c48-8621-fb4388ec66a1/) for the technical report. I will add it.

File: static/stories/sub_select.md (hunk 4 above)
Comment (SarahAlidoost) [reply static/stories/sub_select.md]: added.

File: static/stories/sub_select.md (hunk 2 above)
Comment (SarahAlidoost) [reply static/stories/sub_select.md]: added.

File: static/stories/sub_select.md (hunk 3 above)
Comment (SarahAlidoost) [reply static/stories/sub_select.md:61]: @tepmo42 could you also provide us an explanation of the grey cells in the table?

**Comment by Peter9192** [comment]
I will go ahead and merge this, so it is in place before tomorrow's user workshop. @tepmo42 if you want to make more changes we can open a new PR. Thanks again @SarahAlidoost for the implementation!

**Pull request merged**